dropwizard-example/src/main/java/com/example/helloworld/resources/HelloWorldResource.java

```diff
@@ -5,7 +5,7 @@
 import com.example.helloworld.api.Saying;
 import com.example.helloworld.core.Template;
 import io.dropwizard.jersey.caching.CacheControl;
-import io.dropwizard.jersey.params.DateTimeParam;
+import org.joda.time.DateTime;
```

Comment:
Should we use `java.time.ZonedDateTime` (or a similar class) here?
We'd like to get rid of Joda Time eventually in favor of the `java.time` API.

Reply:
How about an `Optional<io.dropwizard.jersey.jsr310.ZonedDateTimeParam>`?

Reply:
@rhowe Either `*Param` or `Optional<T>` but combining both IMHO doesn't make sense. 😉 

The `*Param` classes are from a time before `Optional<T>` was part of the Java standard library.

Reply:
I see - I was going by the original `Optional<DateTimeParam>` :)

Reply:
OK, I switched it to use `LocalDateParam` and added a little test to `IntegrationTest`.

Reply:
Is there a test if the date parameter is missing or the string “null”? Or an empty string?

Reply:
There isn't, but there weren't any tests around this endpoint at all before.
Happy to add some - I didn't know how complete the tests for `dropwizard-example` were intended to be.